dgg-utilities.user.js

```diff
@@ -130,7 +130,9 @@ const configItems = {
   nukeColor         : new ConfigItem("nukeColor",          "1f1500"),
   mutelinksColor    : new ConfigItem("mutelinksColor",     "120016"),
   customPhrases     : new ConfigItem("customPhrases",      []      ),
+  customPhrasesSoft : new ConfigItem("customPhrasesSoft",  []      ),
   customColor       : new ConfigItem("customColor",        "1f0000"),
+  customSoftColor   : new ConfigItem("customSoftColor",    "680030"),
```

Comment:
Maybe a more muted default color? Just so it's more in line with the others, y'know PepoComfy 
Something like ```#260019``` maybe?